### ISSUE TITLE
Add WCAG 2.2

### ISSUE THREAD

**Issue by mrwweb** [w3c/wai-statements#172]
Now that WCAG 2.2 has been released, it should be added as an option for the "Accessibility standards applied" question.

**Comment by remibetin** [comment]
@mrwweb Thanks. "WCAG 2.2 Level AA" option has been added to the [Accessibility Statement generator tool](https://www.w3.org/WAI/planning/statements/generator/).
